<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MIAO_calculatorClass</class>
 <widget class="QMainWindow" name="MIAO_calculatorClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>314</width>
    <height>392</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MIAO_calculator</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="button_1">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>140</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_2">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>140</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_3">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>140</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_4">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>190</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_5">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>190</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>5</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_6">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>190</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>6</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_8">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>240</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>8</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_7">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>240</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>7</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_9">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>240</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>9</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_0">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>290</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="show_ans">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>291</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
   </widget>
   <widget class="QPushButton" name="button_plus">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>140</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_minus">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>190</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_plot">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>290</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>.</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_equal">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>290</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>=</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_mul">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>240</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>×</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_div">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>290</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>÷</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_del">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>←</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_clear">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>190</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>C</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="button_binary">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>二进制</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="button_octal">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>110</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>八进制</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="button_decimal">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>110</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>十进制</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="button_hex">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>110</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>十六进制</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_menu">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>290</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>Menu1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_power">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>240</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>y^x</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_square">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>140</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>x^2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_cube">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>190</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>x^3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_e">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>290</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>e</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_pi">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>240</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>PI</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>314</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="MIAO_calculator.qrc"/>
 </resources>
 <connections/>
</ui>
